src/routes/game/components/GameOverDialog.vue

```diff
@@ -111,11 +121,14 @@ export default {
     ...mapStores(useGameStore),
     heading() {
       if (this.matchIsOver) {
-        return this.playerWinsMatch ? 'You Win the Match' : 'You Lose the Match';
+        // You win the match / you lose the match
+        return this.playerWinsMatch ? this.t('game.dialogs.gameOverDialog.youWinTheMatch') : this.t('game.dialogs.gameOverDialog.youLoseTheMatch');
```

Comment:
```suggestion
        return this.t(this.playerWinsMatch ?  'game.dialogs.gameOverDialog.youWinTheMatch' : 'game.dialogs.gameOverDialog.youLoseTheMatch' );
```

Reply:
We can dry this up a little bit by putting the conditional inside `this.t()`, we can do this on a few lines here .. 29, 33, 131

Reply:
All set! Thanks for the review @Haviles04